tests/product/tests/data/file_system/file_system_test_cases.py

```diff
@@ -50,9 +50,9 @@
 
 cis_1_1_11 = [
     ('CIS 1.1.11', 'chmod', '0710', '/var/lib/etcd', 'failed'),
-    ('CIS 1.1.11', 'chmod', '0710', '/var/lib/etcd/some_file.txt', 'failed'),
+    # ('CIS 1.1.11', 'chmod', '0710', '/var/lib/etcd/some_file.txt', 'failed'),
     ('CIS 1.1.11', 'chmod', '0600', '/var/lib/etcd', 'passed'),
-    ('CIS 1.1.11', 'chmod', '0600', '/var/lib/etcd/some_file.txt', 'passed'),
+    # ('CIS 1.1.11', 'chmod', '0600', '/var/lib/etcd/some_file.txt', 'passed'),
 ]
```

Comment:
@yashtewari, according to the CIS benchmark - "1.1.11 Ensure that the etcd data directory permissions are set to 700 or more restrictive (Automated)". So commented cases are not relevant and may be removed.

Reply:
Yup, we don't actually need to evaluate the contents of that directory, so they were disabled. Removing as part of the upcoming merge.